### ISSUE TITLE
Problem: radar chart does not maximise available values

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#194]
Problem: currency radar charts for ratings etc do not maximise the available data. A currency's graph data for the wallet ranking (for example) is calculated here: https://github.com/Blockrazor/blockrazor/blob/f42f3d1eb2ba8182b63c4a9fa16c79ead1b716c4/client/normal/currency.js#L48
This means that if a currency has the lowest of all ELO rankings in the system, it can still be shown as a relatively high score on the radar chart if there are no extremely high rankings.

This may become less of a problem over time as larger numbers of currencies are rated multiple times but it's not really possible to predict so I'm adding this to track the problem.

**Comment by anbud** [comment]
@gazhayes, 
If you want to have a nice spread over the graph, we could calculate the graph data like this:
`graphdata(x) = ((x - min) / (max - min)) * 10`

Example:
```
let max = GraphData.findOne({_id: "elodata"}).walletMaxElo
let min = GraphData.findOne({_id: "elodata"}).walletMinElo

let wallet = ((currencyData.walletRanking - min) / (max - min)) * 10
```

**Comment by janat08** [comment]
fixed in #290

**Comment by gazhayes** [comment]
Looks like the wallet data is fixed but not the other data points.

**Comment by janat08** [comment]
Should I also remove hard coded values as in "currencyData.communityRanking || 400" where I assume it's suppose to be minimal values?

**Comment by gazhayes** [comment]
Yes, and you're correct that's just a placeholder for minimum value.

**Comment by janat08** [comment]
fixed in 295

**Comment by janat08** [comment]
Will adjust compare graphs too.
